pkg/operator/config/manager/manager.yaml

```diff
@@ -44,6 +44,8 @@ spec:
               value: ${LOG_LEVEL}
             - name: API_SERVICE_SECURITY
               value: ${API_SERVICE_SECURITY}
+            - name: LISTENING_PORT
```

Comment:
LISTENING_PORT is a bit generic, can we have a more specialized name? Like `ALLOC_API_SVC_PORT`

Reply:
Adjusted name of var